src/shared/buttercup/types.js

```diff
@@ -0,0 +1,6 @@
+export const archiveTypes = {
```

Comment:
Should this perhaps be uppercase `ArchiveTypes`? Just a bit weird seeing it as camel 🙃